drone_localization/streamlit/gallery_app.py

```
@@ -0,0 +1,384 @@
+                        st.success(f"✅ Изображение {delete_id} успешно удалено")
+                    else:
+                        st.error(f"Ошибка: {response.json().get('detail', 'Неизвестная ошибка')}")
+                except Exception as e:
+                    st.error(f"Ошибка: {str(e)}")
+        else:
+            st.warning("Подтвердите удаление")
+
+    st.divider()
+
+st.subheader("☢️ Полная очистка галереи")
+st.error("⚠️ Внимание! Это удалит ВСЕ изображения из галереи без возможности восстановления!")
```

Ruff (0.15.12)

[warning] String contains ambiguous `В` (CYRILLIC CAPITAL LETTER VE). Did you mean `B` (LATIN CAPITAL LETTER B)?

(RUF001)